Firmware/ControladorRobo/ControladorRobo.ino

```diff
@@ -81,23 +80,23 @@ ISR(TIMER1_COMPA_vect)
 {
     // Calculo das velocidades dos motores
 
-    float arcRight = ((float(pulsosDir))/3200.0)*M_PI;
-    float arcLeft = ((float(pulsosEsq))/3200.0)*M_PI;
+    float rightAngularDelta = ((float)right.read()/ENCODER_RESOLUTION)*M_PI;
+    float leftAngularDelta = ((float)left.read()/ENCODER_RESOLUTION)*M_PI;
 
-    wEsq = arcLeft*freq;
-    wDir = arcRight*freq;
+    float rightAngularSpeed = rightAngularDelta*freq;
+    float leftAngularSpeed = leftAngularDelta*freq;
 
-    posDir+=arcRight;
-    posEsq+=arcLeft;
+    rightAngPos += rightAngularDelta;
+    leftAngPos += leftAngularDelta;
```

Comment:
it's not yet. but I intend to use the final angular or linear position to measure rotation angles.